hack/check-task-pipeline-bundle-repos.sh

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/bash -e
+
+SCRIPTDIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+cd "$SCRIPTDIR/.."
+
+QUAY_ORG=redhat-appstudio-tekton-catalog
+
+locate_bundle_repo() {
+    local -r repo_name=$1
+    curl -I -s -L -w "%{http_code}\n" -o /dev/null "https://quay.io/api/v1/repository/${QUAY_ORG}/${repo_name}"
+}
+
+has_missing_repo=
+
+# tasks
+for task_file in task/*/*/*.yaml; do
+    task_name=$(yq e '.metadata.name' "$task_file")
+    found=$(locate_bundle_repo "task-$task_name")
+    if [ "$found" != "200" ]; then
+	echo "Missing task bundle repo: task-$task_name"
+	has_missing_repo=yes
+    fi
+done
+
+# pipelines
+for pl_name in $(oc kustomize pipelines/ | yq e '.metadata.name' | grep -v "^-\+$"); do
```

Comment:
without yq:
```
oc kustomize pipelines/ | oc apply --dry-run=client -f- -o jsonpath='{range .items[*]}{.metadata.name}{"\n"}{end}'
```

Reply:
`oc apply` connects to a OpenShift instance even if using `--dry-run=client`:

```
unable to recognize "STDIN": Get "https://api.ci-ln-0wjh7rb-76ef8.origin-ci-int-aws.dev.rhcloud.com:6443/api?timeout=32s": dial tcp: lookup api.ci-ln-0wjh7rb-76ef8.origin-ci-int-aws.dev.rhcloud.com: no such host
unable to recognize "STDIN": Get "https://api.ci-ln-0wjh7rb-76ef8.origin-ci-int-aws.dev.rhcloud.com:6443/api?timeout=32s": dial tcp: lookup api.ci-ln-0wjh7rb-76ef8.origin-ci-int-aws.dev.rhcloud.com: no such host
unable to recognize "STDIN": Get "https://api.ci-ln-0wjh7rb-76ef8.origin-ci-int-aws.dev.rhcloud.com:6443/api?timeout=32s": dial tcp: lookup api.ci-ln-0wjh7rb-76ef8.origin-ci-int-aws.dev.rhcloud.com: no such host
unable to recognize "STDIN": Get "https://api.ci-ln-0wjh7rb-76ef8.origin-ci-int-aws.dev.rhcloud.com:6443/api?timeout=32s": dial tcp: lookup api.ci-ln-0wjh7rb-76ef8.origin-ci-int-aws.dev.rhcloud.com: no such host
```

Does `registry.redhat.io/openshift4/ose-cli:v4.11` image have default configuration for that?

Reply:
Looks some prerequisition is required for running `oc apply`:

```
resource mapping not found for name: "docker-build" namespace: "" from "STDIN": no matches for kind "Pipeline" in version "tekton.dev/v1beta1"
ensure CRDs are installed first
resource mapping not found for name: "enterprise-contract" namespace: "" from "STDIN": no matches for kind "Pipeline" in version "tekton.dev/v1beta1"
ensure CRDs are installed first
resource mapping not found for name: "java-builder" namespace: "" from "STDIN": no matches for kind "Pipeline" in version "tekton.dev/v1beta1"
ensure CRDs are installed first
resource mapping not found for name: "nodejs-builder" namespace: "" from "STDIN": no matches for kind "Pipeline" in version "tekton.dev/v1beta1"
ensure CRDs are installed first
```

Reply:
yop you need to have Openshift pipelines installed